src/dbcp/etl.py

```diff
@@ -217,6 +217,7 @@ def etl(args):
                 if_exists="append",
                 index=False,
                 schema="data_warehouse",
+                chunksize=1000,
```

Comment:
Did that help performance at all? Not the biggest deal on an ETL process that only takes 3 minutes, but I've always been shocked at how slow df.to_sql() is, no matter what kwargs I use. For my other work I had to make a backend-specific writer that used the `COPY x FROM STDIN WITH CSV` statement. It was literally 1000x faster. I should take the 10 minutes to copy it over here

Reply:
Yeah, I have had the same experience with `df.to_sql()`. I'm not sure if it made an improvement tbh.  I could do some more thorough testing if you like.